### DISCUSSION TITLE
How the get block / section left side highlight?

### DISCUSSION THREAD

**Discussion by mpiorowski** [nvimdev/lspsaga.nvim#479] [Q&A] [answered]
@glepnir  How did you achivied taht orange line on the left?

![image](https://user-images.githubusercontent.com/26543876/187486254-58257d18-6153-4dfd-ae4b-5460fc479d3f.png)

**Comment by glepnir** [comment, marked as answer]
it's indent-blankline plugin.

**Comment by mpiorowski** [comment]
@glepnir thank You :)
